compiler/rustc_codegen_ssa/src/mir/place.rs

```diff
@@ -558,6 +558,9 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                     bug!("using operand local {:?} as place", place_ref);
                 }
             }
+            LocalRef::PendingOperand => {
+                bug!("using still-pending operand local {:?} as place", place_ref);
```

Comment:
This is the one place that wasn't already distinguishing between `Some`/`None`, and near as I can tell that's just because the `None` case is impossible -- at least it seems plausible that it shouldn't ever happen, and building stage2 + running miri tests never hit an ICE here.